File: PAT (Basic-Level)/Python/1001.py
#!/usr/bin/python3
# -*- coding: utf-8 -*-

i = 0
num = int(input())
while num != 1:
    if num % 2 == 1:
        num = (3*num + 1) / 2
        i = i + 1
    else:
        num = num / 2
        i = i + 1
print(i)

File: PAT (Basic-Level)/C/1001.c
#include <stdio.h>

int main(void)
{
	int num, i = 0;
	scanf("%d", &num);
	while(num! = 1){
		if((num % 2) == 1){
			num = (3*num + 1) / 2;
			i++;
		}else{
			num = num/2;
			i++;
		}
	}
	printf("%d", i);
	return 0;
}
